bin/node-bindings/odiff.js

```diff
@@ -47,16 +47,43 @@ function optionsToArgs(options) {
   return argArray;
 }
 
+/** @type {(stdout: string) => Partial<{ diffCount: number, diffPercentage: number }>} */
+function parsePixelDiffStdout(stdout) {
+  const parts = stdout.split(";");
+  
+  if (parts.length === 2) {
+    const [diffCount, diffPercentage] = parts;
+
+    try {
+      return {
+        diffCount: parseInt(diffCount),
+        diffPercentage: parseFloat(diffPercentage),
+      };
+    } catch (e) {
+      console.warn(
+        "For some reason can not parse output from odiff internal process. Please make an issue https://github.com/dmtrKovalenko/odiff/issues/new with stacktrace",
+        e
+      );
+    }
+  }
+
+  return {};
+}
+
 const CMD_BIN_HELPER_MSG =
   "Usage: odiff [OPTION]... [BASE] [COMPARING] [DIFF]\nTry `odiff --help' for more information.\n";
 
 async function compare(basePath, comparePath, diffOutput, options) {
   return new Promise((resolve, reject) => {
     let producedStdout, producedStdError;
 
+    const binaryPath = options.__binaryPath
+      ? options.__binaryPath
+      : path.join(__dirname, "bin", "odiff");
+
     execFile(
-      path.join(__dirname, "bin", "odiff"),
-      [basePath, comparePath, diffOutput, ...optionsToArgs(options)],
+      binaryPath,
+      [basePath, comparePath, diffOutput].concat(optionsToArgs(options)),
```

Comment:
Thats my bad, I was thinking about performance but just tested and realised that concat is actually slower because it stores all the values right in stack which is expensive. Will revert this back